lodqa-ws.rb

```diff
@@ -61,18 +61,24 @@ class LodqaWS < Sinatra::Base
 	end
 
 	get '/simple/login' do
-		redirect Lodqa::Oauth::SIMPLE[:url_auth]
+		session[:mode] = 'simple'
```

Comment:
sessionの名前はグローバルなので、もうちょっと長くて用途が区別しやすいほうが嬉しいです。

```suggestion
		session[:mode_before_login] = 'simple'
```
などはどうでしょうか？

Reply:
確かにそうですね。対応しました。